examples/verify_addresses_from_csv/verify.py

```diff
@@ -0,0 +1,112 @@
+# Usage
+# python verify.py input.csv
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../../..'))
+
+# TODO: Provide your API key, keep this secure
+lob.api_key = '<insert your api key here>'
```

Comment:
I'm not sure how we had it on the other ones, but we should keep it consistent. I think it's `your key here` on the others. I like the idea of `your api key here`.